### PR TITLE
Release v.4.7.0 stable prep

### PR DESCRIPTION
**Pull request by julianrex**
Updates podspecs & changelog for release.

### CODE REVIEW

File: platform/ios/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## master
+
+### Other changes
+
+* Added an `MGLStyle.enablePlacementTransitions` property to control how long it takes for collided labels to fade out. ([#13565](https://github.com/mapbox/mapbox-gl-native/pull/13565))
+* Fixed a crash when casting large numbers in `NSExpression`. ([#13580](https://github.com/mapbox/mapbox-gl-native/pull/13580))
+
+
```
Comment (captainbarbosa) [inline platform/ios/CHANGELOG.md]: On `master` this goes under _Styles & rendering_, and there is no `## master` heading. Should this be the same?

Ref: https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/CHANGELOG.md#styles-and-rendering

Comment (julianrex) [reply platform/ios/CHANGELOG.md]: IIRC specific headings come into play when there are "enough" entries. "Other changes" does sound odd though.

File: platform/ios/CHANGELOG.md [hunk 2]
```diff
@@ -32,7 +39,6 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Added `MGLLoggingConfiguration` and `MGLLoggingBlockHandler` that handle error and fault events produced by the SDK. ([#13235](https://github.com/mapbox/mapbox-gl-native/pull/13235))
 * Fixed random crashes during app termination. ([#13367](https://github.com/mapbox/mapbox-gl-native/pull/13367))
 * Fixed a crash when specifying MGLShapeSourceOptionLineDistanceMetrics when creating an MGLShapeSource. ([#13543](https://github.com/mapbox/mapbox-gl-native/pull/13543))
```
Comment (fabian-guerra) [inline platform/ios/CHANGELOG.md]: `MGLShapeSourceOptionLineDistanceMetrics` & `MGLShapeSource`

Comment (julianrex) [reply platform/ios/CHANGELOG.md]: Have updated in the cherry-pick. We should address this (in master) in a subsequent PR.

**Pull request merged**